javascript/javascript1/week1/week-1.js

```diff
@@ -0,0 +1,73 @@
+//* Age-ify (A future age calculator) *//
+
+const yearOfBirth = 2005;
+const yearFuture = 2076;
+
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+//* Goodboy-Oldboy (A dog age calculator) *//
+
+const dogYearOfBirth = 2022;
+const dogYearFuture = 2034;
+
+const dogAge = dogYearFuture - dogYearOfBirth;
+shouldShowResultInDogYears = false;
+
+//* Acording to Google every human year equals approximately four or five years for a dog :) *//
+
+if (shouldShowResultInDogYears) {
+    console.log(`Your dog will be ${dogAge * 5} dog years old in ${dogYearFuture}.`);
```

Comment:
You are using [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)! 🤩  Well done, this is the easiest way of interpolating varaibles into strings! 👍